jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryingLogGroupsTest.kt

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import com.intellij.openapi.application.runWriteAction
+import com.intellij.openapi.projectRoots.ProjectJdkTable
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.testFramework.IdeaTestUtil
+import com.intellij.testFramework.RunsInEdt
+import com.intellij.testFramework.runInEdtAndWait
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import software.aws.toolkits.jetbrains.utils.rules.JavaCodeInsightTestFixtureRule
+import org.assertj.core.api.Assertions.assertThat
+import software.aws.toolkits.resources.message
+import java.util.Calendar
+import java.util.Date
+
+@RunsInEdt
+class QueryingLogGroupsTest {
+    @JvmField
+    @Rule
+    val projectRule = JavaCodeInsightTestFixtureRule()
+    private val qEditorValidator = QueryEditorValidator
+    private lateinit var view: QueryEditor
+
+    @Before
+    fun setup() {
+        val project = projectRule.project
+        val sdk = IdeaTestUtil.getMockJdk18()
+        runInEdtAndWait {
+            runWriteAction {
+                ProjectJdkTable.getInstance().addJdk(sdk, projectRule.fixture.projectDisposable)
```

Comment:
Why are we getting a mock JDK? If this is fixing something, we are probably doing something incorrect

Reply:
Removed the JDK portion, program working as expected